miniwin/src/internal/d3drmrenderer_citro3d.h

```diff
@@ -68,9 +68,12 @@ class Citro3DRenderer : public Direct3DRMRenderer {
 
 inline static void Citro3DRenderer_EnumDevice(LPD3DENUMDEVICESCALLBACK cb, void* ctx)
 {
-	GUID guid = Citro3D_GUID;
-	char* deviceNameDup = SDL_strdup("Citro3D");
-	char* deviceDescDup = SDL_strdup("Miniwin driver");
+	Direct3DRMRenderer* device = Citro3DRenderer::Create(640, 480);
+	if (!device) {
+		return;
+	}
+	delete device;
+
```

Comment:
As I understand it you already have enough issues with double initilization / shutdown and Citro3D should always work on 3DS.

```suggestion
```